frontend/src/metabase/dashboard/components/Dashboard/Dashboard.jsx

```diff
@@ -356,6 +271,7 @@ class Dashboard extends Component {
               >
                 <DashboardHeader
                   {...this.props}
+                  isNavbarOpen={isNavbarOpen}
```

Comment:
```suggestion
```

This is unnecessary as `DashboardHeader` already gets `isNavbarOpen` from redux

https://github.com/metabase/metabase/blob/a10cfbc0ec88de53967b638701a0e64e774d7fd6/frontend/src/metabase/dashboard/containers/DashboardHeader.jsx#L55

Reply:
yes, but long time ago spreading props was considered as a harmful behaviour and I'd love not to search for it next time, but see it exactly passed

Reply:
You're right, this is pretty much the same thing as before. If we want to keep this, can we remove `isNavBarOpen` inside `DashboardHeader` to avoid confusion then?

Reply:
I didn't get, `isNavBarOpen` is used inside DashboardHeader, I just wanted to pass the prop explicitly

Reply:
[As stated above ](https://github.com/metabase/metabase/pull/35358#discussion_r1381681524), `DashboardHeader` now gets `isNavBarOpen` from:
- prop and
- redux

So both are redundant, we only need one. I propose we remove the prop injected from redux
https://github.com/metabase/metabase/blob/a10cfbc0ec88de53967b638701a0e64e774d7fd6/frontend/src/metabase/dashboard/containers/DashboardHeader.jsx#L55

Reply:
removed prop passing